packages/lib-subject-viewers/src/VolumetricViewer/models/ModelAnnotations.js

```diff
@@ -82,6 +82,16 @@ export const ModelAnnotations = ({ onAnnotation }) => {
             annotations: annotationModel.annotations
           })
         },
+        inactive: () => {
```

Comment:
[minor][🤡] One of the very weird/niche/nitpicky things that threw me for a loop here is that ".inactive" (and ".active()", come to think of it) is an _adjective,_ whereas all the other functions in the ModelAnnotations model are _verbs._ ("add()", "remove()", "undo()", "publish()", etc) My brain is mentally renaming this to ".deactivate()", which sounds more verb-y and also makes me think we're running a starship.

This is just me rambling though, so _please don't actually change the code, it's fine as is._